docs/management/connectors/action-types/jira.asciidoc

```diff
@@ -37,7 +37,7 @@ Name::      The name of the connector.
 URL::       Jira instance URL.
 Project key:: Jira project key.
 Email::  The account email for HTTP Basic authentication.
-API token::  Jira API authentication token for HTTP Basic authentication.
+API token::  Jira API authentication token for HTTP Basic authentication. **For Jira Data Center this should be the password matching the username above.**
```

Comment:
I don't see a username field, so I'm assuming it's the email account:
```suggestion
API token::  Jira API authentication token for HTTP Basic authentication. For Jira Data Center, this value should be the password associated with the email owner.
```

Reply:
Should the UI text also be changed to "API token or password"?

Reply:
@adcoelho mentioned the same but the password is only valid for Jira Data Center and the API token only valid for Jira Cloud. It may consume some cloud users that they can use their password instead of an API token. @adcoelho created this issue https://github.com/elastic/kibana/issues/199104 to enhance the form.